### DISCUSSION TITLE
False Positive Zone tags

### DISCUSSION THREAD

**Discussion by szaske** [blakeblackshear/frigate#1563] [Ask A Question]
I'm trying to figure out why Frigate would determine this event is in my driveway. He does have a long shadow, but even that is  not close to my driveway. Any clues?  Do shadows play a role in zone detection? I've seen those ML detections that can determine the skeletal shape of the person. It would be very cool if Frigate could detect feet, and then only announce the location of the feet...which is really what I want to know.  In this example the person is in the street zone. His shadow is in the sidewalk zone, and his head is in the far_side zone. In this event he was tagged as Road and Driveway, which would seem impossible to not include the sidewalk zone.

<img width="848" alt="frigate-zone" src="https://user-images.githubusercontent.com/19757740/130112046-638689ba-9ba9-48c6-ac37-46fafcbe1bed.png">
<img width="848" alt="false-driveway" src="https://user-images.githubusercontent.com/19757740/130112059-ad7b2554-effb-4916-875f-593c27173b3c.png">

**Comment by MrAnonymous722** [comment]
From my understanding, it should base the objects location on the bottom center of the box it draws. So not sure why that's considered to be in your driveway.

**Comment by blakeblackshear** [comment]
Probably similar to this issue: https://github.com/blakeblackshear/frigate/issues/1246

**Reply by blakeblackshear** [reply]
You may be able to use the min_area filter on the zone to prevent false positives from triggering the zone.

**Reply by szaske** [reply]
is min_area the widthxheight of the bounding box?  So the required size of the bounding box to fire a detection?

**Reply by blakeblackshear** [reply]
Yes, you can see the values next to the % confidence score in the bounding box text.
